### ISSUE TITLE
[Fixed in GH-6052] Safari: Missing visual focus and cannot tab on Checkout button

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#5038]
<img width="1147" alt="Screen Shot 2019-10-16 at 1 48 05 PM" src="https://user-images.githubusercontent.com/45409515/66944676-a7b5bf00-f01b-11e9-87e7-4221f97872f5.png">

**Comment by KateChuen** [comment]
Still an issue

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-31 at 4 16 08 PM](https://user-images.githubusercontent.com/15113219/73574969-11c86780-4445-11ea-8889-60095e01e47d.png)

**Comment by KateChuen** [comment]
Tested on dev3
- Chrome ✅
- Firefox ✅
- Safari ✅
